### ISSUE TITLE
Texture of the dogs mismatch

### ISSUE THREAD

**Issue by transbiod** [DashieDev/DoggyTalentsNext#9]
Hi, i use latest version of the mod and one of the dog textures repeats two times. Because of that i have one more dog with broken texture that i can not replace. Can you please help me with that? I will provide everything that you need to fix this problem.
![image](https://user-images.githubusercontent.com/41019733/221835800-5ddd9023-0b8c-42c2-8a94-6e3c911c02a9.png)
It's doggytex16

**Comment by DashieDev** [comment]
Could you please describe the problem in more detail? What are you trying to achieve? What do you mean by "broken texture that i cannot replace"?  What did you do before? 
You may try restarting the game meanwhile.

**Comment by transbiod** [comment]
Ok, sorry i should have added more details. The problem is in the styles section. One of the dogs textures appears there two times. I think that's because one texture is being used by two dogs at the same time. Also i still have one dog that doesn't have texture despite me replacing all of the files in the folder.
![Снимок экрана_20230223_204449](https://user-images.githubusercontent.com/41019733/221910196-57fd8014-a58e-4d84-a04d-aa6433819401.png)
![Снимок экрана_20230223_204540](https://user-images.githubusercontent.com/41019733/221910233-3b6644c1-5bbf-4fff-9137-476757935357.png)
This is from regular version without me changing the files. After changing them i learned that doggytex16 appears to be the problematic one
English isn't my mother language, so i tend to write strangely, sorry in advance!

**Comment by DashieDev** [comment]
Great, I will be looking into it. Thank you very much for reporting! ❤️

**Comment by DashieDev** [comment]
I seems to have a bit trouble reproducing it... Could you please provide exactly how your instance went to that state ? Like what was you doing before that ?  Which skin did you select? Did you replace any skin ? (i can see that you have the missing skins populated but i wonder if you override any existing skin ?)

**Comment by transbiod** [comment]
It was like that when I installed this mod. Can it somehow interact with entity texture features mod? I will delete all the wolf random textures from my texturepack and test it without them.

**Comment by DashieDev** [comment]
Could i see your dog textures folder content (the one in your screenshot)? If you are okay with it, you may upload the content of the folder here. https://drive.google.com/drive/folders/1g--XwIGApqnoM-7EfawRuIUnCDS6o3xC

**Comment by transbiod** [comment]
Ok, will do

**Comment by DashieDev** [comment]
Also did you previously have the original (Percivalalb's version) loaded there? If so did the problem occurs while you are using the version? 
And also do you have any other mod installed?

**Comment by transbiod** [comment]
And also, deleting random wolf textures didn't help. Well, at least i tried!(

**Comment by transbiod** [comment]
Done uploading

**Comment by DashieDev** [comment]
Okay i got it i will test on my instance with your textures tomorrow, thanks ❤️. Gotta sleep for school tmr :)

**Comment by transbiod** [comment]
No, i installed your mod first. And yes i have 400+ mods, so testing will be long and difficult. But i have another mod that interacts with wolves and dogs that i can point to https://www.curseforge.com/minecraft/mc-mods/revampedwolf

**Comment by transbiod** [comment]
> Okay i got it i will test on my instance with your textures tomorrow, thanks ❤️. Gotta sleep for school tmr :)

Ok, have a nice sleep!

**Comment by DashieDev** [comment]
Okay i may have know why there is still one texture missing from the selection view when you replace all of them, it may because DT thought "override.json" was a texture file but it is not, it is a bug that is going on for a while now, that also explains why if you look in the log, you would see DT occasionally throws an exception which stated briefly as "override.json" is not a texture file (which is harmless and not cause a crash). 
Also does repetition happens after you replaces all of them ?

**Comment by transbiod** [comment]
Hi, i will test it today with all of the textures replaced

**Comment by transbiod** [comment]
Ok, i thought that replacing all of the textures with the same one except doggytex16 will be the easiest way to do it, but i never thought that this would be the result.
![image](https://user-images.githubusercontent.com/41019733/222712664-e5bb99be-b568-464f-ad33-4e2dc832c7ef.png)

**Comment by transbiod** [comment]
A have only two textures and a missing one because of the override bug, but no repeting ones!
